packages/@aws-cdk/aws-ivs/lib/channel.ts

```diff
@@ -0,0 +1,172 @@
+   *
+   * @attribute
+   */
+  readonly channelArn: string;
+
+  /**
+   * Adds a stream key for this IVS Channel.
+   * @param id construct ID
+   */
+  addStreamKey(id: string): StreamKey;
+}
+
+/**
+ * Reference to a new or existing IVS Channel
+ */
+abstract class ChannelBase extends core.Resource implements IChannel {
+  public abstract readonly channelArn: string;
+
+  public addStreamKey(id: string): StreamKey {
+    return new StreamKey(this, id, {
+      channel: this,
+    });
+  }
+}
+
+/**
+  Channel latency mode
+*/
+export enum LatencyMode {
+  /**
+   * Use LOW to minimize broadcaster-to-viewer latency for interactive broadcasts.
+   */
+  LOW = 'LOW',
+
+  /**
+   * Use NORMAL for broadcasts that do not require viewer interaction.
+   */
+  NORMAL = 'NORMAL',
+}
+
+/**
+  * The channel type, which determines the allowable resolution and bitrate.
+  * If you exceed the allowable resolution or bitrate, the stream probably will disconnect immediately.
+*/
+export enum ChannelType {
+  /**
+   * Multiple qualities are generated from the original input, to automatically give viewers the best experience for
+   * their devices and network conditions.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ivs-channel.html
+   */
+  STANDARD = 'STANDARD',
+
+  /**
+   * delivers the original input to viewers. The viewer’s video-quality choice is limited to the original input.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ivs-channel.html
+   */
+  BASIC = 'BASIC',
+}
+
+/**
+ * Properties for creating a new Channel
+ */
+export interface ChannelProps {
+  /**
+   * Whether the channel is authorized.
+   *
+   * If you wish to make an authorized channel, you will need to ensure that
+   * a PlaybackKeyPair has been uploaded to your account as this is used to
+   * validate the signed JWT that is required for authorization
+   *
+   * @default false
+   */
+  readonly authorized?: boolean;
+
+  /**
+   * Channel latency mode.
+   *
+   * @default LatencyMode.LOW
+   */
+  readonly latencyMode?: LatencyMode;
+
+  /**
+   * Channel name.
+   *
+   * @default - None
+   */
+  readonly name?: string;
+
+  /**
+   * The channel type, which determines the allowable resolution and bitrate.
+   * If you exceed the allowable resolution or bitrate, the stream will disconnect immediately
+   *
+   * @default ChannelType.STANDARD
+   */
+  readonly type?: ChannelType;
+}
+
+/**
+  A new IVS channel
+*/
+export class Channel extends ChannelBase {
+  /**
+   * Import an existing channel
+   */
+  public static fromChannelArn(scope: Construct, id: string, channelArn: string): IChannel {
+    // The astute among you may notice that this regex differs from the documentation
+    // This is because the documentation is wrong :)
+    let channelArnRegex = /^arn:aws:ivs:[a-z0-9-]+:[0-9]+:channel\/[a-zA-Z0-9-]+$/;
```

Comment:
this regex is not partition aware (i.e. `aws-cn` is a valid partition that will not work if the service becomes available in that region in the future)

Reply:
I'm not sure if we should change this if the API side validates it as shown. I'll check with the team

Reply:
leaving this unresolved for now - if we decide we don't want it to be partition aware, we should add a comment and justification for why it's appropriate here

Reply:
I've changed this in the latest commit to just use Arn.parse